src/assets/images/404.svg

```diff
@@ -1,21 +1,14 @@
-<path fill-rule="evenodd" clip-rule="evenodd" d="M288.237 93.179L257.837 93.179C256.953 93.179 256.237 92.4483 256.237 91.5469L256.237 13.2108C256.237 12.3095 256.953 11.5788 257.837 11.5788L288.237 11.5788C289.12 11.5788 289.837 12.3095 289.837 13.2108L289.837 91.5469C289.837 92.4483 289.12 93.179 288.237 93.179Z" fill="#1AA9A5"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M251.437 13.1795L251.437 43.5795C251.437 44.4632 250.706 45.1795 249.805 45.1795L171.469 45.1795C170.567 45.1795 169.837 44.4632 169.837 43.5795L169.837 13.1795C169.837 12.2958 170.567 11.5795 171.469 11.5795L249.805 11.5795C250.706 11.5795 251.437 12.2958 251.437 13.1795Z" fill="#92C3E6"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M417.434 13.1587H442.767C443.504 13.1587 444.101 13.8034 444.101 14.5987V83.7187C444.101 84.514 443.504 85.1587 442.767 85.1587H417.434C416.698 85.1587 416.101 84.514 416.101 83.7187V14.5987C416.101 13.8034 416.698 13.1587 417.434 13.1587Z" fill="#1AA9A5"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M444.101 121.159V142.064C444.101 142.669 443.601 143.159 443.101 143.159L417.101 143.159C416.601 143.159 416.101 142.669 416.101 142.064L416.101 121.254C416.101 120.649 416.601 120.159 417.101 120.159L443.101 120.159C443.601 120.159 444.101 120.554 444.101 121.159Z" fill="#105351"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M365.826 85.336L351.837 85.336C350.894 85.336 350.264 84.3626 350.652 83.5023L381.836 14.2468C382.008 13.8646 382.296 13.5462 382.659 13.3365L382.799 13.2555C382.996 13.1411 383.221 13.0809 383.449 13.0809L410.037 13.0809C410.981 13.0809 411.61 14.0549 411.222 14.9153L379.806 84.5885C379.595 85.0558 379.128 85.3556 378.616 85.3542C371.97 85.336 367.746 85.336 365.826 85.336Z" fill="#92C3E6"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M459.673 91.6591L459.673 113.659C459.673 114.317 459.305 115.159 458.101 115.159L352.101 115.159C350.897 115.159 350.673 114.626 350.673 113.968L350.673 91.6591C350.673 91.0016 350.897 90.1586 352.101 90.1586L458.101 90.1586C459.305 90.1586 459.673 91.0016 459.673 91.6591Z" fill="#508197"/>
-</g>
-<defs>
-<clipPath id="clip0">
-<rect width="459.673" height="143.159" fill="white"/>
-</clipPath>
-</defs>
+<svg width="440" height="139" viewBox="0 0 440 139" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M23.511 85.3361L7.51096 85.3361L39.4461 15.9895C40.2626 14.2167 42.0359 13.081 43.9877 13.081L69.7339 13.081L37.1447 85.3566C30.011 85.3361 25.511 85.3361 23.511 85.3361Z" fill="#92C3E6"/>
+<path d="M12.5215 113C9.76009 113 7.52151 110.761 7.52151 108L7.52151 88H111.522C114.283 88 116.522 90.2386 116.522 93V108C116.522 110.761 114.283 113 111.522 113H12.5215Z" fill="#508197"/>
+<path d="M100.522 116V134C100.522 136.761 98.2829 139 95.5215 139H77.5215C74.7601 139 72.5215 136.761 72.5215 134V116H100.522Z" fill="#105351"/>
```

Comment:
really nice fix. Great logic and implementation

Reply:
^^^ this is a joke

Reply:
<3